src/control/cmd/agent/main.go

```diff
@@ -57,17 +57,69 @@ type cliOptions struct {
 	Insecure   bool       `short:"i" long:"insecure" description:"have agent attempt to connect without certificates"`
 	RuntimeDir string     `short:"s" long:"runtime_dir" description:"Path to agent communications socket"`
 	LogFile    string     `short:"l" long:"logfile" description:"Full path and filename for daos agent log file"`
+	NetworkDbg bool       `short:"n" long:"netdebug" description:"Enable extended network device scan debug output"`
```

Comment:
Just a suggestion, but I'm wondering, what if we made the debug param take an optional argument instead? Could be a default level and a "net" option.

Reply:
It's a neat idea.  I am not sure how to add that into the cliOptions correctly though.  Do you have in mind a way to do that?

Reply:
It seems like the most straightforward way would be to make the --debug option take an optional string argument. Something like:
```
	Debug string `short:"d" long:"debug" optional:"1" optional-argument:"basic" choice:"basic" choice:"net"`
```

This would let people still invoke it with the -d option - if they did it would turn on the basic debugging.

Reply:
Thanks.  I got it going with a minor tweak:

> Debug      string     `short:"d" long:"debug" optional:"1" optional-value:"basic" choice:"basic" choice:"net"`